mne/gui/_viewer.py

```diff
@@ -122,20 +126,11 @@ class Object(HasPrivateTraits):
     scene = Instance(MlabSceneModel, ())
     src = Instance(VTKDataSource)
 
-    color = Color()
-    rgbcolor = Property(depends_on='color')
+    color = Tuple()
```

Comment:
When I click on the view options button in the coreg gui I get something that might be related:

```
Exception occurred in traits notification handler for object: <mne.gui._coreg_gui.CoregFrame object at 0x121275230>, trait: view_options, old value: <undefined>, new value: 0
Traceback (most recent call last):
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traits/trait_notifiers.py", line 340, in __call__
    self.handler( *args )
  File "/Users/christian/Code/mne-python/mne/gui/_coreg_gui.py", line 1480, in _view_options_fired
    self.view_options_panel.edit_traits()
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traits/has_traits.py", line 1871, in edit_traits
    handler, id, scrollable, args )
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/view.py", line 433, in ui
    ui.ui( parent, kind )
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/ui.py", line 221, in ui
    self.rebuild( self, parent )
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/toolkit.py", line 155, in ui_live
    ui_live.ui_live( ui, parent )
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_live.py", line 39, in ui_live
    _ui_dialog(ui, parent, BaseDialog.NONMODAL)
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_live.py", line 59, in _ui_dialog
    BaseDialog.display_ui(ui, parent, style)
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_base.py", line 482, in display_ui
    ui.owner.init(ui, parent, style)
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_live.py", line 177, in init
    self.add_contents(panel(ui), bbox)
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_panel.py", line 255, in panel
    panel = _GroupPanel(content[0], ui).control
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_panel.py", line 592, in __init__
    layout = self._add_items(content, inner)
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_panel.py", line 860, in _add_items
    editor.prepare(inner)
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/editor.py", line 158, in prepare
    self.update_editor()
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/instance_editor.py", line 319, in update_editor
    self.resynch_editor()
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/instance_editor.py", line 380, in resynch_editor
    self.factory.id )
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/view.py", line 433, in ui
    ui.ui( parent, kind )
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/ui.py", line 221, in ui
    self.rebuild( self, parent )
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/toolkit.py", line 141, in ui_subpanel
    ui_panel.ui_subpanel( ui, parent )
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_panel.py", line 76, in ui_subpanel
    _ui_panel_for(ui, parent, True)
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_panel.py", line 82, in _ui_panel_for
    ui.control = control = _Panel(ui, parent, is_subpanel).control
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_panel.py", line 140, in __init__
    self.control = panel(ui)
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_panel.py", line 255, in panel
    panel = _GroupPanel(content[0], ui).control
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_panel.py", line 592, in __init__
    layout = self._add_items(content, inner)
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/qt4/ui_panel.py", line 860, in _add_items
    editor.prepare(inner)
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/editor.py", line 156, in prepare
    self.init( parent )
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/editors/tuple_editor.py", line 94, in init
    self._ts     = ts = TupleStructure( self )
  File "/Users/christian/anaconda/envs/mne-dev/lib/python2.7/site-packages/traitsui/editors/tuple_editor.py", line 203, in __init__
    enter_set = enter_set ) )
TypeError: 'NoneType' object is not callable
```

Looking at the file from the last line (`tuple_editor.py`) it looks like what gets called is `type` which comes from something called editor factory. So it looks to me like a bug in `traitsui`. Or is it necessary to pre-specify the type of the entries of the tuple (it looks like the view should add different widgets depending on the types http://code.enthought.com/projects/traits/docs/html/TUIUG/factories_basic.html#tupleeditor)?